src/lib/ledger_catchup/ledger_catchup.ml

```diff
@@ -4,6 +4,27 @@ open Protocols.Coda_transition_frontier
 open Pipe_lib
 open Coda_base
 
+(** [Ledger_catchup] is a process that connects a foreign external transition
+    into a transition frontier by asking it's peer a path of external_transitions.
+    It receives the external_transition from [Catchup_monitor]. With that
+    external_transition, it will ask it's peers for a path of
+    external_transitions from it's root to the transition it is asking for. It
+    will then do some verifications on these breadcrumbs. If all the
```

Comment:
external_transition != breadcrumb

`It will then validate and build breadcrumbs from these external transitions`